.jhipster/GloablSettings.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Is this file a duplicate from `.jhipster/GlobalSettings.json` with a typo?

Reply:
Der `.jhipster`-Ordner ist autogeneriert, daher würde ich dort nichts ändern